parser_test.go

```diff
@@ -407,6 +407,19 @@ const exampleStateFile = `
 							"tags.Role": "worker"
 						}
 					}
+				},
+				"linode_instance.eighteen": {
+					"type": "linode_instance",
+					"depends_on": [],
+                    "primary": {
```

Comment:
Can you please tidy up the indentation here? It uses tabs, for whatever reason.

Reply:
Of course, I've tidied that up to match existing formatting